### PR TITLE
[npud] Add Logging class and related config files

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit adds Logging class and related config files.
It turns the log on and off using `NPUD_LOG_ENABLE` environment.

ONE-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

Related issue: #9304

### CODE REVIEW

**Comment by hseok-oh** [comment]
Tizen security check doesn't allow to use environment variable. So runtime tizen build disabled this feature.

https://github.com/Samsung/ONE/blob/c5f3f873364b93ec59b7b63566c1ea857fb68b46/infra/nnfw/cmake/options/options_armv7l-tizen.cmake#L9-L11

Runtime still uses environment variable because NNAPI doesn't have way to config runtime without environment variable. IMHO, we need to find better way to config `npud`.

**Comment by jyoungyun** [comment]
/cc @hseok-oh 
What about this changes? https://github.com/Samsung/ONE/pull/9509/commits/2acb05f1eedd2722b0df403f39aa606d90644435

File: packaging/nnfw.spec
Comment (hseok-oh) [inline packaging/nnfw.spec:156]: Please define different option variable. (ex. `option_debug`?)
BTW, Don't we need to use logging on release build?

Comment (jyoungyun) [reply packaging/nnfw.spec]: I added `option_config` define. Could you see this patch? https://github.com/Samsung/ONE/commit/02f9c350b9903cc5a7edbed4dab5e9d461e609a6

```
# Set option for configuration
%define option_config %{nil}
%if %{config_support} == 1
%if %{npud_build} == 1
# ENVVAR_NPUD_CONFIG: Use environment variable for npud configuration and debug
%define option_config -DENVVAR_NPUD_CONFIG=ON
%endif # npud_build
%endif # config_support
```

**Review by hseok-oh — APPROVED** [review]
This PR has some minor issue, but conceptually LGTM.
I'll check minor issue after merging this PR.
- Code duplication
- Useless code: copied from runtime but some used for nnfw API

File: runtime/service/npud/core/Server.cc
Comment (ragmani) [inline runtime/service/npud/core/Server.cc:36]: I have a question. Why did you change `std::endl` to `\n`? Was there a problem?

Comment (jyoungyun) [reply runtime/service/npud/core/Server.cc:36]: No, there is no problem. I just follow different code rules.
One runtime uses both of `"blah blah ~~\n"` and `"blah bla~~" << std::endl`.

**Comment by jyoungyun** [comment]
/cc @hseok-oh 

> I'll check minor issue after merging this PR.
> - Code duplication
> - Useless code: copied from runtime but some used for nnfw API

Thank you!

File: runtime/service/npud/core/Server.cc
Comment (ragmani) [reply runtime/service/npud/core/Server.cc:36]: I was wondering if there was a problem. I'm happy because we would have to change the other codes like this in `onert` if there was a problem. But I think that `std::endl` is better than `\n` because `std::endl` flushes the stream. Anyway this change doesn't matter.

**Review by ragmani — APPROVED** [review]
LGTM

**Pull request merged**
